docs/QSG/docker.md

```diff
@@ -1,212 +1,102 @@
 
 - The `daos-server` node running a DAOS server daemon managing data storage devices such as SCM or
   NVMe disks.
-- The `daos-admin` node allowing to manage the DAOS server thanks to `dmg`command.
-- The `daos-client` node using the the DAOS server to store data.
+- The `daos-admin` node is used to manage the DAOS server thanks to the `dmg`command.
+- The `daos-client` node uses the DAOS server to store data.
 
-At this time only emulated hardware storage are supported by this Docker platform:
+At this time only emulated hardware storage is supported by this Docker platform:
 
-- SCM (i.e. Storage Class Memory) are emulated with standard RAM memory.
+- SCM (i.e. Storage Class Memory) is emulated with standard RAM memory.
 - NVMe disks are emulated with a file device.
 
-!!! warning
-    Virtual Docker network such as [bridge](https://docs.docker.com/network/bridge/) are not yet
-    well supported by DAOS.  Thus, one physical network interface (i.e. loopback interface is also
-    not well supported) of the host should be chosen for being used by the containers through the
-    Docker [host](https://docs.docker.com/network/host/) network.
-
-
-## Prerequisites
-
-To build and deploy the Docker images, `docker` and optionally `docker-compose` shall be available.
-The docker host should have access to the [Docker Hub](https://hub.docker.com/) and
-[Rocky Linux](https://rockylinux.org/) official repositories.  Finally,
-[hugepages](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) linux kernel feature shall
-be enabled on the docker host.  At least, 4096 pages of 2048kB should be available.  The number of
-huge pages allocated could be checked with the following command:
-
-```bash
-$ sysctl vm.nr_hugepages
+**!!!Warning**
+DAOS support of virtual networking is still in development. The following are known issues:
+- [bridge](https://docs.docker.com/network/bridge/) 
+- loopback
+	- Workaround - use one physical network interface for the [host](https://docs.docker.com/network/host/) used by the Docker containers network.
+
+## Hardware and Software Prerequisites
+### Hardware Prerequisites
+- 5GB of DDR and 
+- 16GB of storage.
+
+### Software Prerequisites
+- BIOS
+	- Enable VT-d 
+- To build and deploy the Docker images, `docker` 
+	- optionally `docker-compose`
+- Hugepages must be enabled
+	- At least, 4096 pages of 2048kB should be available 
+- Internet access for the following repositories
+	- [Docker Hub](https://hub.docker.com/) 
+	- [Rocky Linux](https://rockylinux.org/) official repositories.  
+	- [hugepages](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) enabled
+		-Linux kernel feature shall be enabled on the docker host. At least, 4096 pages of 2048kB should be available. 
+
+### Hardware and software configuration used in this QSG
+- Server Board: Wolf Pass
+- CPU: Cascade Lake (2)
+- OS: Ubuntu 20.0.4LTE. fresh Install
+- Memory: 192GB 
+- PMEM:(12)x16GB (not required for this QSG)
+- BIIOS: VT-d enable
```

Comment:
since I have stripped out anything from the bash readouts that would need explanation, I can remove, but I would recommend adding back if feed back makes a case of describing the HW that displays x data points in a bash readout. 

I am removing this whole sections "Hardware and software configuration needed for this QSG",